samples/ErniAcademy.Events.Samples/SampleConfiguration.cs

```diff
@@ -0,0 +1,52 @@
+﻿using ErniAcademy.Events.ServiceBus.Extensions;
+using ErniAcademy.Events.StorageQueues.Extensions;
+using ErniAcademy.Events.Redis.Extensions;
+using ErniAcademy.Serializers.Contracts;
+using ErniAcademy.Serializers.Json;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace ErniAcademy.Events.Samples;
+
+public static class SampleConfiguration
+{
+    public static void ConfigureServiceBus(IServiceCollection services, IConfiguration configuration)
+    {
```

Comment:
Instead commenting the code please add a correct ///<summary> with all the information